website/docs/r/network_watcher_flow_log.html.markdown

```diff
@@ -0,0 +1,120 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_network_watcher_flow_log"
+sidebar_current: "docs-azurerm-resource-network-watcher-flow-log"
+description: |-
+  Manages a Network Watcher Flow Log.
+
+---
+
+# azurerm_network_watcher_flow_log
+
+Manages a Network Watcher Flow Log.
```

Comment:
Is it worth noting anything here about how we can't delete it and are just disabling it when we call destroy?

Reply:
We are not "deleting" it in the traditional sense but we do clear the FIL property which is the same as deletion i think